### ISSUE TITLE
Remove visual settings

### ISSUE THREAD

**Issue by Flamefire** [Return-To-The-Roots/s25client#350]
I'd remove the "visual" settings of the warehouses, as they are possibly wrong (probably the reason for #349)

The "visual" showing of the settings before it is actually changed is a good idea, but they should be kept in sync with the actual setting. Assume e.g. a GameCommand is dropped (can happen during pause, maybe we should remove this restriction as GCs are not send during pause) then the visual setting will always be the opposite of the actual setting.

So 2 ideas:
1. Remove the visual setting altogether and just toggle the image in the ingame window on a click -> Reopening the window will always show the actual status, but that might be outdated a few ms later if one happens to open-reopen it fast -> wh might notify the window so it can update the overlays
2. Add an event on every change of the visual settings with a duration of 2.5 times the NWF length and set the visual settings equal to real settings on event. The event is replaced by another change of the visual settings and changes to the real setting also change the visual setting and remove the event --> window needs to be notified, but settings are most of the times correct (GC received at right time, GC ignored, GC received later)

I'd opt for option 1. What do you thing? @jhkl @MarcusSt @Flow86

**Comment by jhkl** [comment]
I might be wrong, but aren't the visual settings also used for some ingame (not in a window) elements like the "Do not supply with coins" sign? This would not work with option 1 I guess?

**Comment by Flamefire** [comment]
Right. For those cases we would need option 2. I'd also replace all toggle game commands by explicit enable/disable GCs so that on execution the actual state is correctly updated. This would already solve some problems: Disabling a coin supply that is disabled but visually enabled would do nothing but update the visual state.

**Comment by Spikeone** [comment]
Just asking, this is not about removing them from the game, right?
Sounded title sounds like you want to remove the icons indicating the current state.

**Comment by Flamefire** [comment]
No. The point is: You have visual and actual settings. Once you set e.g. coins stop, then visually the coins are disabled, a command is send to disable them, and when that is executed they are actually disabled. Problem is: Coins are not enabled or disabled but toggled: On<->Off. So assume this: Stop coins-> Show disabled sign + send command. Command never gets executed -> Coins show as stopped but are actually enabled. If you try to enable them, they are actually disabled and so on...

**Comment by Flamefire** [comment]
Improved to a good state for now, final fix can be done later
